articles/azure-app-configuration/push-kv-devops-pipeline.md

```diff
@@ -19,6 +19,7 @@ The [Azure App Configuration Push](https://marketplace.visualstudio.com/items?it
 - App Configuration resource - create one for free in the [Azure portal](https://portal.azure.com).
 - Azure DevOps project - [create one for free](https://go.microsoft.com/fwlink/?LinkId=2014881)
 - Azure App Configuration Push task - download for free from the [Visual Studio Marketplace](https://marketplace.visualstudio.com/items?itemName=AzureAppConfiguration.azure-app-configuration-task-push).
+- Latest agent version (for [self-hosted agents](https://docs.microsoft.com/en-us/azure/devops/pipelines/agents/agents?view=azure-devops&tabs=browser#install)).
```

Comment:
We need more details than this.

Reply:
How about something like, "Latest [vsts-agent ](https://github.com/microsoft/azure-pipelines-agent/releases) version ( when running the task on [self-hosted agent](https://docs.microsoft.com/en-us/azure/devops/pipelines/agents/agents?view=azure-devops&tabs=browser#install))" ?
Should we also update this [doc](https://docs.microsoft.com/en-us/azure/azure-app-configuration/pull-key-value-devops-pipeline)?